### DISCUSSION TITLE
local linux account auth support along with linux groups

### DISCUSSION THREAD

**Discussion by johnnyq** [filebrowser/filebrowser#1047] [Ideas]
**Is your feature request related to a problem? Please describe.**
The ability for users to login using their local Linux logins with group support would be fantastic.

**Describe the solution you'd like**
Maybe a solution that would sync local linux users to json auth

**Comment by polic72** [comment]
This is almost a make or break feature for me. I really want to be able to log which users are uploading which files in a shared directory at the OS level.

**Comment by ramiresviana** [comment]
The authentication hook can be used to implement custom authentication logic, check https://github.com/filebrowser/filebrowser/pull/1554
